.gitignore

```diff
@@ -1,9 +1,5 @@
 # .gitignore
 
-# Ignore package locks other than Yarn.
```

Comment:
You can replace this with `yarn.lock` in case someone accidentally does a `yarn install`